app/src/main/java/com/eveningoutpost/dexdrip/Services/LibreWifiReader.java

```diff
@@ -58,6 +61,7 @@
     private static final Gson gson = JoH.defaultGsonInstance();
 
     private final static long DEXCOM_PERIOD = 300000;
+    private static OkHttpClient httpClient = null;
```

Comment:
Java object references are initialized to `null`, automatically.

Reply:
Done.

Reply:
Done.